codegen/src/main/ops/org/nd4j/codegen/ops/Linalg.kt

```diff
@@ -0,0 +1,112 @@
+package org.nd4j.codegen.ops
+
+import org.nd4j.codegen.api.DataType
+import org.nd4j.codegen.api.Language
+import org.nd4j.codegen.api.doc.DocScope
+import org.nd4j.codegen.dsl.*
+
+fun Linalg() =  Namespace("Linalg") {
+    //val namespaceJavaPackage = "org.nd4j.linalg"
+
+    Op("Cholesky") {
+        javaPackage = "org.nd4j.linalg.api.ops.impl.transforms"
+        javaOpClass = "Cholesky"
+        Input(DataType.NDARRAY, "input") { description = "Input tensor with inner-most 2 dimensions forming square matrices" }
+        Output(DataType.NDARRAY, "output"){ description = "Transformed tensor" }
+
+        Doc(Language.ANY, DocScope.ALL){
+            """
+             Computes the Cholesky decomposition of one or more square matrices.
+            """.trimIndent()
+        }
+    }
+
+    Op("Lstsq") {
+        javaPackage = "org.nd4j.linalg.api.ops.custom"
+        javaOpClass = "Lstsq"
+
+        Input(DataType.NDARRAY, "matrix") {description = "input tensor"}
+        Input(DataType.NDARRAY, "rhs") {description = "input tensor"}
+        Arg(DataType.FLOATING_POINT, "l2_reguralizer") {description = "regularizer"}
+        Arg(DataType.BOOL, "fast") {description = "fast mode, defaults to True"}
```

Comment:
add default value:
`Arg(DataType.BOOL, "fast") {description = "fast mode, defaults to True"; defaultValue = true}`

Also fix input/output datatypes: should be numerical or floating point (check). "NDArray" means anything, including strings, boolean, etc

Reply:
Fixed.